src/main/java/bio/terra/service/snapshotbuilder/query/Concept.java

```diff
@@ -0,0 +1,44 @@
+package bio.terra.service.snapshotbuilder.query;
+
+public class Concept extends TableVariable {
+
+  public static final String TABLE_NAME = "concept";
```

Comment:
I think a goal here would be to try and make these all `private`, it'd be good to look at each one and try and see if that can be done or not.

In the case of field names, using aliases for each result should let us avoid needing to reference them outside this class.